[Compute AG]-[PCSUP-10975]

Pubs-MV opened the pull request:
Clarification that the the ‘project’ field that you see under the ci scan results will always present the jenkins job name.

Pubs-MV committed:
[Compute AG]-[PCSUP-10975]

Clarification that the the ‘project’ field that you see under the ci scan results will always present the jenkins job name.

tallieber commented:
I think that it still misleading, because this field in the request can be a value of a tenant project. Regarding the “project” column in the scan result view we always display the Jenkins pipeline name

Pubs-MV committed:
Updated based on chat with Tal

We have 2 issues to clarify here :
1. the ‘project’ column in the ci image scan table is the Jenkins job name and this column will have value only when the vendor is jenkins - we will think of another name to this column
2. the project field in the script - this field is related to the “project” feature we have on console  https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-01/prisma-cloud-compute-edition-admin/deployment_patterns/projects (central and tenant project names) and in-order the request to success the user need to give there valid project name he configured.

tallieber commented on compute/admin_guide/continuous_integration/jenkins_pipeline_project.adoc:
```diff
@@ -188,6 +190,7 @@ pipeline {
                 key: '',
                 logLevel: 'info',
                 podmanPath: '',
+                // The project field below is only applicable if you are using Prisma Cloud Compute Edition and have set up projects (multiple consoles) on Prisma Cloud. This field is not the name of the Jenkins pipeline.
```
I think this is enough "The project field below is only applicable if you are using Prisma Cloud Compute Edition and have set up projects (multiple consoles) on Prisma Cloud." The second sentence will just confused them

Pubs-MV commented:
Thank you @tallieber . Can you please approve to merge?

tallieber committed:
Update jenkins_pipeline_project.adoc
